### PR TITLE
fix(Reco): Send empty tracking string instead of undefined

### LINKED ISSUE #95: Send empty tracking string instead of undefined
Send tracking_string: '' with the first call getRecomendation()

### PR DESCRIPTION
**Pull request by artdaw**
Send tracking_string: '' with the first call getRecomendation()

Fix #95

### CODE REVIEW

**Comment by artdaw** [comment]
👍

File: src/api/utils.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const mergePlainOptsWithDefault = (opts = {}, defaultPlainOpts = {}, pathToPlain = null) => {
```
Comment (ricardomatias) [inline src/api/utils.js]: Please add documentation regarding the options

File: src/api/__tests__/utils_tests.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    {
+      cdn: 'cdn1',
+      resolutions: ['small']
+    },
+    'media');
+
+  t.is(cdn, 'cdn2');
+  t.deepEqual(resolutions, ['small']);
+});
+
```
Comment (ricardomatias) [inline src/api/__tests__/utils_tests.js]: Please add a test case with a `pathToPlain` that is not valid.

Comment (artdaw) [inline src/api/__tests__/utils_tests.js]: Done

File: src/api/utils.js (hunk 1 above)
Comment (artdaw) [inline src/api/utils.js]: Done

**Comment by artdaw** [comment]
since it's out of stock I close it

**Pull request closed without merging**